File: tests/conformance/fixtures/go/08-k8s.go
package main

import sykli "github.com/yairfalse/sykli/sdk/go"

func main() {
	p := sykli.New()

	p.Task("train").Run("python train.py").
		K8s(sykli.K8sOptions{Memory: "32Gi", CPU: "4", GPU: 2})

	p.Emit()
}

File: tests/conformance/fixtures/rust/08-k8s.rs
use sykli::{Pipeline, K8sOptions};

fn main() {
    let mut p = Pipeline::new();
    p.task("train").run("python train.py").k8s(K8sOptions { memory: Some("32Gi".into()), cpu: Some("4".into()), gpu: Some(2) });
    p.emit();
}
